store.go

```diff
@@ -2595,6 +2595,41 @@ func copyStringInterfaceMap(m map[string]interface{}) map[string]interface{} {
 
 const configFile = "/etc/containers/storage.conf"
 
+// ThinpoolOptionsConfig represents the "storage.options.thinpool"
+// TOML config table.
+type ThinpoolOptionsConfig struct {
+	// Device specifies a custom block storage device to use for
+	// the thin pool.
+	Device string `toml:"device"`
```

Comment:
What's this option? What does it do? I will take a block device, create lvm thin pool out of it?